go/vm/lfvm/instructions.go

```diff
@@ -749,6 +749,28 @@ func opExtcodehash(c *context) {
 	}
 }
 
+func createShanghaiInitCodeSizeCheck(c *context, size *uint256.Int) bool {
```

Comment:
This reads like the function would create a check ... rename to `checkInitCodeSize`?